openslides/agenda/static/templates/agenda/item-csv-import.html

```diff
@@ -0,0 +1,39 @@
+ <h1 translate>Import agenda items</h1>
+
+<div id="submenu">
+  <a ui-sref="agenda.item.list" class="btn btn-sm btn-default">
+    <i class="fa fa-angle-double-left fa-lg"></i>
+    <translate>Back to overview</translate>
+  </a>
+</div>
+
+<p translate>Select a CSV file to import agenda items!
+
+<p translate>Please note:</p>
```

Comment:
Have you tested this? If the closing p-tag is really required, then you have to add one in line 10